boards/arm/google_kukui/google_kukui_defconfig

```diff
@@ -0,0 +1,33 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Zephyr Kernel Configuration
+CONFIG_ARM=y
+CONFIG_SOC_SERIES_STM32F0X=y
+
+# Platform Configuration
+CONFIG_SOC_STM32F098XX=y
+CONFIG_BOARD_GOOGLE_KUKUI=y
+
+# Serial Drivers
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+
+# Enable console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# Pinmux Driver
+CONFIG_PINMUX=y
+
+# GPIO Controller
+CONFIG_GPIO=y
+
+# Clock configuration
+CONFIG_CLOCK_CONTROL=y
+
+# 48MHz system clock
+CONFIG_CORTEX_M_SYSTICK=y
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=48000000
```

Comment:
HSI48 is not supported by clock_controller driver so far.
So, by activating `CONFIG_CLOCK_STM32_SYSCLK_SRC_HSI=y`, you get 'basic' HSI (8MHz).

Reply:
Ack. Will leave as is for when HSI48 is supported.

Reply:
I'm fine with letting 48MHz as a comment, but please update config to a upstream supported value.
Leaving this opens the door to clock issues on peripherals.